trello_items.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import json
+
+key = "<your trello key>"
+token = "<your trello token>"
```

Comment:
You could try adding these as environment variables (by adding new variables to .env.template and .env, and then loading them with `os.environ.get('YOUR_VARIABLE_NAME')` in your python code. It would be easier to commit changes to trello_items.py without any risk of committing the secrets.

(It would also enable you to deploy the code to a server without making uncommitted code changes)